autobotAI_integrations/handlers/task_handler.py

```
@@ -76,8 +76,17 @@ def handle_task(task: PayloadTask) -> TaskResult:
     else:
         result.resources = formatted_result
     
+    logger.info(f"TASK OUTPUT SELECTOR: {getattr(task, 'output_selector', 'N/A')}")
     if result.resources and hasattr(task, "output_selector") and task.output_selector:
-        result.resources = extract(result.resources, task.output_selector, logger)
+        logger.info(f"BEFORE EXTRACT: {result.resources}")
+        logger.info(f"OUTPUT SELECTOR: {task.output_selector}")
+        try:
+            result.resources = extract(result.resources, task.output_selector, logger)
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)